examples/demo/README.md

```diff
@@ -0,0 +1,31 @@
+# test-vue-file-upload
+
+In this project we have implemented `vue-file-upload` library. so, you can use this code or refer this implementation for your project.
```

Comment:
This ready-to-use example shows real-time usage of the `vue-file-upload` library with custom UI.